src/main/java/com/adyen/model/checkout/details/KlarnaDetails.java

```diff
@@ -0,0 +1,282 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Java API Library
+ *
+ * Copyright (c) 2020 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+
+package com.adyen.model.checkout.details;
+
+import com.adyen.model.checkout.PaymentMethodDetails;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Objects;
+
+import static com.adyen.util.Util.toIndentedString;
+
+/**
+ * KlarnaDetails
+ */
+
+public class KlarnaDetails implements PaymentMethodDetails {
+
+    /**
+     * Possible types
+     */
+    public static String KLARNA = "klarna";
+    public static String KLARNA_PAYMENTS = "klarnapayments";
+    public static String KLARNA_PAYMENTS_ACCOUNT = "klarnapayments_account";
+    public static String KLARNA_PAYMENTS_B2B = "klarnapayments_b2b";
+    public static String KLARNA_PAY_NOW = "klarna_paynow";
+    public static String KLARNA_ACCOUNT = "klarna_account";
+    public static String KLARNA_B2B = "klarna_b2b";
```

Comment:
Travis checkstyle is failing, probably because of this
```suggestion
    public static final String KLARNA_B2B = "klarna_b2b";
```

Reply:
fixed